### PR TITLE
adds command to rollback a release of an app

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

File: src/commands/heroku.rs [hunk 1]
```diff
@@ -169,6 +169,38 @@ pub fn get_app_releases(ctx: &mut Context, msg: &Message, mut args: Args) -> Com
...
+    });
+
+    msg.reply(
+        ctx,
+        match response {
+            Ok(_response) => format!(
+                "App {} was successfully rolled back to the code at {}",
+                app_name, version_to_rollback_to
+            ),
+            Err(e) => format!("An error occured when trying to rollback your app:\n{}", e),
```
Comment (smarnach) [inline src/commands/heroku.rs:197]: I don't think `rollback` is a verb, so I'd add a space there.

File: README.md [hunk 2]
```diff
@@ -177,6 +177,19 @@ Version: 3
...
+**~rollback_app**
+
+If you would like to rollback your app to the code associated with a previous release of your app, you can do so with the ~rollback_app command.
+
+```
+you: ~get_app_releases testing-nell-bot version-to-rollback-to
+```
+
+```
+you: ~get_app_releases testing-nell-bot v5
```
[the remark this answers is not in the document]
Comment (nellshamrell) [reply README.md:189]: It accepts with the v and without the v

File: README.md [hunk 3]
```diff
@@ -177,6 +177,19 @@ Version: 3
 Status: succeeded
 ```
 
+**~rollback_app**
+
+If you would like to rollback your app to the code associated with a previous release of your app, you can do so with the ~rollback_app command.
+
+```
+you: ~get_app_releases testing-nell-bot version-to-rollback-to
```
[the remark this answers is not in the document]
Comment (nellshamrell) [reply README.md:185]: Goodness gracious! Copy pasta!

File: src/commands/heroku.rs (hunk 1 above)
Comment (nellshamrell) [reply src/commands/heroku.rs]: Fixed!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nellshamrell) [reply README.md]: Added clarification to the README on this point.

**Comment by smarnach** [comment]
@nellshamrell Thanks for the updates! Piedro and me both approved the PR, so feel free to merge.

**Pull request merged**